.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+### Support reference number
+<!-- Reference to JIRA ticket issue if available. -->
```

Comment:
We probably shouldn't encourage colleagues to post JIRA links on open source projects? this doesn't make sense right now 😛 
cc @wizzy25

Reply:
@adnasa This has been discussed severally. The JIRA links will not be accessible to the public. And not all issues have accompanying JIRA references

Reply:
i don't that really justifies it. my point is that JIRA shouldn't even openly linked in the first place.

Reply:
but if you guys really need this and it's fine with your PO, then all good from my side